synopsys-task/src/synopsys-task/tools-parameter.ts

```diff
@@ -519,6 +532,13 @@ export class SynopsysToolsParameter {
       covData.data.coverity.network = { airGap: true };
     }
 
+    // Set arbitrary (To support both Coverity and Polaris)
+    covData.data.coverity = Object.assign(
+      {},
+      this.setCoverityArbitraryArgs() as CoverityConnect,
```

Comment:
Seems like `this.setCoverityArbitraryArgs() as CoverityConnect` here  `as CoverityConnect` type Assertion is not required. even if you want to use, this function returns **CoverityArbitrary** not **CoverityConnect**

Reply:
coverity.build.command="mvn clean install"
coverity.clean.command="mvn clean"

As per bridge documentation, it should be under coverity not under coverity.connect.
Same applies to blackduck as well.